models/lobby.go

```diff
@@ -269,6 +282,17 @@ func (lobby *Lobby) IsPlayerReady(player *Player) (bool, *helpers.TPError) {
 	return slot.Ready, nil
 }
 
+func (lobby *Lobby) UnreadyAllPlayers() {
+	var slots []*LobbySlot
```

Comment:
You can just do 
db.Model(&LobbySlot{}).Where("lobby_id = ?", lobby.ID).Update("ready", false)